WD_LOGIN PAGE_ABANOB MIKHAEL

ABANOBMIKHAEL opened the pull request:
Fixed
1. The  the color palette
2. The text box size
3. navbar

adrianweerasinghe97 commented on next_webapp/public/styles/licensing.css:
looks good

adrianweerasinghe97 commented on next_webapp/public/styles/login.css:
I would recommend getting rid of the comments following the css. you could add the comment above the css if necessary

adrianweerasinghe97 commented on next_webapp/src/app/login/page.tsx:
This code seems to have several issues, including duplicate declarations, inconsistent code structures, and nested blocks that should not be nested.

adrianweerasinghe97 reviewed (commented):
1. The licensing.css looks good
2. For the login.css you  could get rid of the comments following the css. you could add the comment above the css if necessary
3. For the page.tsx, the code seems to have several issues, including duplicate declarations, inconsistent code structures, and nested blocks that should not be nested.

ABANOBMIKHAEL commented:
CSS, I removed the comments.
Login(Page.tsx)
1.Removed Duplicate Declarations
2.Consistent Code Structure
Please let me know if there is anything else.
Thank you

kaseyuu commented:
Hi Abanob, could you please resolve the conflicts?
